modules/virt-gathering-virt-data.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * virt/logging_events_monitoring/virt-collecting-virt-data.adoc
+
+[id="virt-gathering-virt-data_{context}"]
+= Gathering targeted virtualization data
+
+When you use the `must-gather` CLI tool to collect information about {VirtProductName}, only `VirtualMachine` and `VirtualMachineInstance` custom resources (CRs) are gathered by default. You can request additional or targeted data by including parameters when you run the `must-gather` command.
+
+To customize the data that `must-gather` collects, append a double dash (`--`) to the command, followed by one or more parameters. For example:
+
+[source,terminal,subs="attributes+"]
+----
+$ oc adm must-gather --image=registry.redhat.io/container-native-virtualization/cnv-must-gather-rhel8:v{HCOVersion}
+ -- <example_parameter>
+----
+
+.Available parameters
+
+`gather_vms_details`:: Collect virtual machine log files, VM definitions, and namespaces (and their child objects) that belong to {VirtProductName} resources. If you use this parameter without specifying a namespace or VM, the `must-gather` tool collects this data for all VMs in the cluster.
+
+`gather_images`:: Collect image and image stream custom resource information.
+
+`NS=`:: Gather virtual machine details from the namespace that you specify.
+
+`VM=`:: Gather details about a particular virtual machine. To use this option, you must also specify a namespace by using the `NS=<namespace>` parameter.
+
+`PROS=`:: Modify the maximum number of parallel processes that the `must-gather` tool uses. By default, the tool uses no more than five parallel processes.
++
+[IMPORTANT]
```

Comment:
Can we a structure similar to the original PR? This are not regular command line parameters, and the order is important. 

For example, `NS`, `VM` or `POS` are environment variable and should appear before the script name.

`gather_images` or `gather_vms_details` are alternative scripts for the default one (`gather`). But even adding `PROS=7` to the default script, override it and you'll need to explicitly add it, e.g. `oc adm must-gather --image... -- PROS=7 gather`

Reply:
Thank you for this info! In the original PR, there is an example that uses the `PROS` env variable but does not specify `gather` as the script. I expanded the usage information to reflect this and included an example that uses the `gather` script. Please review it when you can.